spec/griddler/cloudmailin/adapter_spec.rb

```diff
@@ -47,4 +124,25 @@ def default_params
       EOS
     }
   end
+
+  def bcc_params
+    default_params.merge(
+      envelope: default_params[:envelope].merge(to: 'sandra@example.com')
+    )
+  end
+
+  def nocc_params
+    default_params.merge(
+      headers: default_params[:headers].except(:Cc)
+    )
+  end
+
+  def cc_params
+    default_params.merge(
+      headers: default_params[:headers].merge(
+        Cc: 'Some Identifier <some-identifier@example.com>',
+        To: 'emily@example.com'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.